### PR TITLE
Add funeral director company Roc-Eclerc

### PR DESCRIPTION
**Pull request by bmillemathias**
https://overpass-turbo.eu/s/HrU

### CODE REVIEW

File: brands/shop/funeral_directors.json [hunk 1]
```diff
@@ -9,5 +9,13 @@
       "name": "The Co-operative Funeralcare",
       "shop": "funeral_directors"
     }
+  },
+  "shop/funeral_directors|Roc-Eclerc": {
+    "tags": {
+      "brand": "Roc-Eclerc",
+      "brand": "Q62558102",
```
Comment (matkoniecz) [inline brands/shop/funeral_directors.json]: It should be rather `brand:wikidata`.

Comment (bmillemathias) [reply brands/shop/funeral_directors.json]: indeed.

**Comment by matkoniecz** [comment]
"Roc Eclerc" is the same but using incorrect version of name, right? https://overpass-turbo.eu/s/HrW

In that case adding `match` would be a good idea - see https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#card_file_box--about-the-brand-files

**Comment by bmillemathias** [comment]
> 
> 
> "Roc Eclerc" is the same but using incorrect version of name, right? https://overpass-turbo.eu/s/HrW
> 
> In that case adding `match` would be a good idea - see https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#card_file_box--about-the-brand-files

I did a check before submittting the PR and Roc-Eclerc is the correct name in the french registry of companies so I wanted to correct "Roc Eclerc" in "Roc-Eclerc" in OSM but sure, I can do that also.

**Comment by matkoniecz** [comment]
Adding `"match"` property will help with exactly that - it is used to mark different names as an incorrect variants.

See https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md#card_file_box--about-the-brand-files

`amenity=fast_food name=McDonald's` is a correct version, `amenity=fast_food name=Mc Donald's` and some other are recognized as matching variants.

**Comment by matkoniecz** [comment]
Thanks for a PR!

**Pull request merged**